storage/Devices/BcacheImpl.cc

```diff
@@ -53,17 +54,29 @@ namespace storage
 
```

Comment:
Do the TODOs above still apply?

Reply:
I think yes, except cache mode.

Reply:
so? hint hint

Reply:
Yes, the others still apply. But that does not block this PR.

Reply:
I meant to remove those that are addressed in this PR, which has been done.